### DISCUSSION TITLE
how to enable n_cores in MDAnalysis.analysis.encore

### DISCUSSION THREAD

**Discussion by meghaparashar2** [MDAnalysis/mdanalysis#4823] [Q&A] [answered]
Dear all,
I am having two trajectories on which I want to perform the ensemble similarity calculations using harmonic and cluster ensemble similarity methods. I want to use the utility n_cores here, but even after providing a value greater than 1, it still is using 1 CPU core. Can anyone please help me with this?

Thanks,
Megha

**Comment by meghaparashar2** [comment]
Dear @RMeli, @orbeckst, @hmacdope 
I would greatly appreciate any help here.

Best regards,
Megha

**Reply by orbeckst** [reply]
@enoee can you help with this ENCORE question?

**Comment by enoee** [comment, marked as answer]
Hi Megha.
According to https://github.com/MDAnalysis/mdaencore/issues/27#issuecomment-1776893690 parallelization only applies to multiple set of parameters and hence the described behavior could be normal?
